js/apps/admin-ui/src/dashboard/Dashboard.tsx

```diff
@@ -67,48 +69,46 @@ const EmptyDashboard = () => {
   );
 };
 
+const FeatureItem = ({ feature }: { feature: FeatureRepresentation }) => {
```

Comment:
Can we move this into a seperate type?
```suggestion
type FeatureItemProps = {
  feature: FeatureRepresentation
};

const FeatureItem = ({ feature }: FeatureItemProps) => {
```